_pages/ato/walkthrough.md

```diff
@@ -45,21 +45,25 @@ The controls that are _not_ inherited from an underlying system must be listed i
 
-### Step 5 — Authorize the system
+### Step 5 — Complete documentation package
 
-The full list of data and functions in and of the system (in government parlance "mission based information types" and "delivery mechanisms") must be itemized in structured data. While the data types are obviously arbitrary and custom to each system we produce, the government has a formalized data set of mission functions that should be mapped to the system via NIST 800-60. For a Rails app, for example, this can simply be a link to the `db/schema.rb` file on GitHub.
+Fill out the documentation in the checklist. We hope to auto-create the SSP from the Compliance Masonry YAML file at some point, but for now you should copy and paste.
 
-Fill out the [System Security Plan (SSP)](../ssp/) and link from your project's [checklist](../checklist/) issue. The designated AO will conduct a quality assurance check on your materials.
+The full list of data and functions in and of the system (in government parlance "mission based information types" and "delivery mechanisms") must be itemized in structured data. While the data types are obviously arbitrary and custom to each system we produce, the government has a formalized data set of mission functions that should be mapped to the system via [NIST 800-60](http://csrc.nist.gov/groups/SMA/fisma/categorization.html). For a Rails app, for example, this can simply be a link to the `db/schema.rb` file on GitHub.
 
-You should send materials to the Authorizing Official (AO) as soon as the security boundary of the system is fixed and implemented (i.e. none of the [categorization](../levels) is changing). This is a judgement call. If your are uncertain as to your system's security boundary, you should consult with 18F Infrastructure early in your development process. If your system is providing novel or risky functions, or handling extremely sensitive data, you should consult with 18F Infrastructure as early as possible.
+### Step 6 — Authorize the system
 
-#### Signing an authorization
+Your Infrastructure Lead will work with you to schedule and prioritize your system assessment. Once assessment starts, the first step is that the Authorizing Official will review all the items in your ATO checklist including all the documents you generated. When the items in the ATO checklist are ready, your Infrastructure Lead will circulate the Rules of Engagement document for signatures. 
 
-Once all of the materials are prepared and the system is considered "ready" by the Authorizing Official, they will sign the ATO.
+Then, a team with members from the project team, TTS Infrastructure, your Infrastructure Lead, and the GSA OCISO will convene for at least a week as the ATO Sprinting Team, and begin to follow the [Lightweight Security Authorization Process](https://insite.gsa.gov/portal/content/627230). 
```

Comment:
Might want to link to [the README](https://docs.google.com/document/d/1bGOV_pp_BlAzZsoa2D5pnsinx3R2gVnaBdFvHkwv0ig/edit).